app/Http/Controllers/Bible/BiblesController.php

```diff
@@ -1031,9 +1031,10 @@ public function annotations(Request $request, $bible_id)
             return $this->setStatusCode(401)->replyWithError(trans('api.projects_users_not_connected'));
         }
 
-
         $book_id = checkParam('book_id');
         $chapter = checkParam('chapter');
+        $chapter_max_verses = 180;
+        $limit   = (int) (checkParam('limit') ?? $chapter_max_verses);
```

Comment:
we should cap the limit passed in on the request. if the user passes in a number greater than chapter_max_verses, we want the limit to be set to chapter_max_verses.